apple/RNSVGRenderable.h

```diff
@@ -36,6 +42,10 @@
 @property (nonatomic, assign) CGPathRef hitArea;
 @property (nonatomic, strong) NSString *filter;
 
+#ifdef RCT_NEW_ARCH_ENABLED
+- (facebook::react::RNSVGRenderableShadowNode::ConcreteState::Shared)state;
+#endif
+
```

Comment:
It's only internal property, we don't need to define it here

Reply:
It is also used inside `RNSVGGroup.mm`